source/wp-content/themes/wporg-parent-2021/sass/block-styles.scss

```diff
@@ -222,7 +222,7 @@
 	line-height: var(--wp--custom--body--short-text--typography--line-height);
 }
 
-.is-style-with-arrow {
+.is-style-with-arrow a[href]:first-of-type {
```

Comment:
The `first-of-type` selector prevents multiple arrows with multiple anchors. The layout behaviour can be a little unexpected if the anchor is used on a sub string of the heading, but I'm not sure this is worth trying to mitigate:

![Screen Shot 2022-08-22 at 2 31 48 PM](https://user-images.githubusercontent.com/1017872/185827275-ab8d8b14-0c6c-48b8-a212-4e3be9619272.jpg)

Reply:
I agree, we don't need to address that issue now. If it becomes something site editors need to do, we can figure it out then.